_posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Partner Highlight: Pureinsights adds OpenSearch to its Discovery Platform"
+authors:
+  - plewis
+date: 2022-04-15 08:50:01
```

Comment:
Please do a quick update to the date to 2022-04-22

thanks

Reply:
PR-775 has the required changes 
Thank you